src/webui/api/appcontroller.cpp

```diff
@@ -749,6 +751,9 @@ void AppController::setPreferencesAction()
     // Validate HTTPS tracker certificate
     if (hasKey("validate_https_tracker_certificate"))
         session->setValidateHTTPSTrackerCertificate(it.value().toBool());
+    // Block peer connections on privileged ports
+    if (hasKey("no_connect_privileged_ports"))
```

Comment:
Using the same key `block_peers_on_privileged_ports` should make it less surprising for webapi users.

Reply:
Sure, done.